src/grid/api.service.ts

```diff
@@ -86,8 +88,25 @@ export class IgxGridAPIService {
         this.get(id).filteringExpressions = filteringState;
     }
 
+    public filter_multiple(id: string, expressions: IFilteringExpression[]) {
```

Comment:
Why are there methods not camel-cased?

Reply:
Those are internal from the point of the grid and are not exposed directly to the user.

Reply:
Then why is `filterGlobal` camel-cased?